src/ai/ai_gui.cpp

```diff
@@ -310,8 +310,6 @@ struct AISettingsWindow : public Window {
 		this->vscroll = this->GetScrollbar(WID_AIS_SCROLLBAR);
 		this->FinishInitNested(slot);  // Initializes 'this->line_height' as side effect.
 
-		this->SetWidgetDisabledState(WID_AIS_RESET, _game_mode != GM_MENU && Company::IsValidID(this->slot));
```

Comment:
What is this change doing here?

Reply:
https://github.com/OpenTTD/OpenTTD/pull/7890#issue-358508040

Reply:
ok... why? Does it work? Why is it a good idea?

Reply:
Because otherwise it is disabled.
![Unnamed, 1951-02-23](https://user-images.githubusercontent.com/43006711/72679914-0885e500-3aac-11ea-9f08-09f06666e0dd.png)
It works!
![Unnamed, 1951-03-04](https://user-images.githubusercontent.com/43006711/72679915-091e7b80-3aac-11ea-9a3e-5bf79f432f70.png)
Because AIs have this button disabled while in a game.